src/components/select/Select.js

```diff
@@ -61,10 +62,14 @@ const Select = ({
                             <span className="select-overlay" />
                         </span>
                         {infoTooltip && (
-                            <Tooltip position="middle-right" text={infoTooltip}>
-                                <span className="tooltip-icon-container">
```

Comment:
is there any chance that components using this could be using `span` to add override styles? preferably not, but i'm wondering if this counts as a breaking change